workloads/ingress-perf/run.sh

```diff
@@ -10,9 +10,11 @@ CONFIG=${CONFIG:?}
 BASELINE_UUID=${BASELINE_UUID:-}
 BASELINE_INDEX=${BASELINE_INDEX:-ingress-performance-baseline}
 TOLERANCY=${TOLERANCY:-20}
+OS=$(uname -s)
```

Comment:
I have noticed some CI containers are super minimal, we might see failures with `uname`. 

i have used 
```
export OS=${OS:-Linux}
export ARCH=$(arch)
```
In the past and know it "worksforme" 😄

Reply:
They  belong to the same RPM

```shell
rsevilla@wonderland $ rpm -qf /usr/bin/arch
coreutils-9.1-8.fc37.x86_64
rsevilla@wonderland  $ rpm -qf /usr/bin/uname
coreutils-9.1-8.fc37.x86_64

```